cfgov/jinja2/v1/_includes/organisms/image-text-50-50-group.html

```diff
@@ -1,3 +1,30 @@
+
+{# ==========================================================================
+
+   Image and Text 50/50 Group
+
+   ==========================================================================
+
+   Description:
+
+   See [GHE]/CFPB/hubcap/wiki/50-50-Info-unit-group
+
+   Create an info-unit-group organism when given:
+
+   value: Object defined from a StreamField block.
+
+   value.heading: A string containing title of module.
+
+   value.image_texts: TODO: add description for this.
+
+   value.link_image_and_heading: TODO: add description for this.
+
+   value.sharing.shareable: TODO: add description for this.
+
+   value.sharing.share_blurb: TODO: add description for this.
+
+   ========================================================================== #}
+
```

Comment:
This one is also deprecated and could point toward `info-unit-group-2.html`.

Reply:
Makes sense, thanks for the background!

Reply:
Added deprecated note.